### DISCUSSION TITLE
Permitir que el Botón seleccionado de un RadioButtonGroup pueda cambiar su appearence.

### DISCUSSION THREAD

**Discussion by sbocaccio** [TiendaNube/nimbus-design-system#195] [Q&A] [answered]
Buenas! 

Estoy queriendo usar el `RadioButtonGroup` y poder elegir el valor deseado. Me gustaría poder mostrar de alguna forma cual es la opción seleccionada (cambiando la apariencia como se ve en la imagen) pero estoy pudiendo lograrlo. Mirando la interfaz de `RadioButtonGroup` y en particular la de `InterfaceRadioButtonOption` no veo forma de realizarlo.

Existe alguna forma de lograr esto?

![Captura de Tela 2023-10-12 às 11 36 17](https://github.com/TiendaNube/nimbus-design-system/assets/98828814/7c1078e8-4772-408e-9259-e3285703569f)

**Comment by juniorconquista** [comment, marked as answer]
@sbocaccio en nuestra versión más nueva de la biblioteca de componentes nimbus esse comportamento ja é o padrão o ideal seria você utilizar a versión más reciente una vez que vamos a descontinuar em breve esa versión de dos componentes antiguos. Deixo o [enlace](https://nimbus.nuvemshop.com.br/documentation/atomic-components/radio) da documentação desse componente.
